db/mongo.go

```diff
@@ -0,0 +1,87 @@
+package db
+
+import (
+	"log"
+
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+//Technology collects the slack user ID of people who use or want to learn a
+//given technology, identified by Name
+type Technology struct {
+	ID       string   `json:"id"`
+	Name     string   `json:"name"`
+	Users    []string `json:"users"`
+	Learners []string `json:"learners"`
+}
+
+func getSession() *mgo.Session {
+	session, err := mgo.Dial("mongodb://localhost")
```

Comment:
As a slackbot, I guess it would be used quite intermittently. Will look more closely at whether one slack command would be likely to make more than one call to the database and make a decision :)